flyteadmin_config.yaml

```diff
@@ -206,3 +206,5 @@ qualityOfService:
     # by default production has an UNDEFINED tier when it is omitted from the configuration
 namespace_mapping:
    template: "{{ project }}-{{ domain }}" # Default namespace mapping template.
+admin:
+  endpoint: "dns:///localhost:30081"
```

Comment:
Why is this needed? Also when running in local mode this  portwont work since we dont bring up ingress and instead would require 8088 /8089

Reply:
yes removed it for now